django_elastic_migrations/urls.py

```diff
@@ -1,9 +1,8 @@
 # -*- coding: utf-8 -*-
+from __future__ import (absolute_import, division, print_function, unicode_literals)
```

Comment:
This line isn't necessary for python3 any longer....though it does not harm at all to leave it.

Reply:
Until we're ported 100%, I was planning to support both 2 and 3, so I'll leave it in for py2's sake for now.